### PR TITLE
Add tags, universities, deparments and titles endpoints

### PR DESCRIPTION
**Pull request by FarukOzderim**
Tags, universities, deparments and titles endpoints are added. Please review the additions. #226

### CODE REVIEW

**Comment by EnesToptas** [comment]
- Not everyfield should be returned as output
- The new tables in the database should also have unique integer ids, that would simplify operations
- Besides get, post/put endpoints also should be implemented

**Review by EnesToptas — CHANGES_REQUESTED** [review]
After adding posts you can merge

**Comment by FarukOzderim** [comment]
requested add endpoints are added, you can review & merge.

**Comment by hazerb** [comment]
No need to put result of database query into array. but anyway, in overall seems fine.

File: backend/controllers/autoCompleteController.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        res.status(500).send(error)
+    }
+}
+
+getTitles = async function (req, res) {
+    try {
+        titles = await Title.findAll({attributes: ['title']})
+        var array=[]
+        
+        for(var i in titles)
```
Comment (EnesToptas) [inline backend/controllers/autoCompleteController.js:68]: pls fix that, we don't need arrays. that would make the code significantly better.

**Review by EnesToptas — APPROVED** [review]
Overall seems fine, you can merge it, or if you want, you could change the unnecessary array part.

File: backend/controllers/autoCompleteController.js (hunk 1 above)
Comment (FarukOzderim) [reply backend/controllers/autoCompleteController.js:68]: I talked with the frontend and they prefered the output as array.

Comment (EnesToptas) [reply backend/controllers/autoCompleteController.js:68]: Output is already an array

Comment (FarukOzderim) [reply backend/controllers/autoCompleteController.js:68]: No, the original output format is like this:

image

However front end team preferred an output like this.

image

Comment (EnesToptas) [reply backend/controllers/autoCompleteController.js:68]: Ok then, thanks

**Pull request merged**